check_upgrade.sh

```diff
@@ -0,0 +1,57 @@
+#!/bin/bash
```

Comment:
this assumes current branch has been compiled

Reply:
Maybe you could add some checks for that, e.g. that the `target` folder exists.